test/test_contra_env_setup.py

```diff
@@ -37,73 +52,68 @@ def setUp(self):
         self.vm.args.extend(['-cpu', 'host'])
         self.vm.args.extend(['-accel', 'kvm'])
 
-        # Headless
+        # headless
         self.vm.args.extend(['-display', 'none', '-vga', 'none'])
 
-        # Run!
+        # run!
         self.vm.launch()
 
-        # Wait VM to be UP
+        # wait VM to be UP
         with GetConsole(self.vm) as console:
             console.sendline('sudo resize2fs /dev/sda1')
 
 
     def test(self):
-        distro = self.params.get('distro', default='Fedora')
-
-        # Set the SSH options and port
+        """
+        contra-env-setup test
+        """
+        # set the SSH options and port
         env = {'ANSIBLE_CONFIG': 'test/ansible.cfg'}
         ssh_args = ['-o UserKnownHostsFile=/dev/null',
                     '-o StrictHostKeyChecking=no']
 
-        # Install python, git and update nettle to libvirt network
-        if distro == 'Fedora':
-            cmd = (' ansible localhost -i "localhost," --become '
-                ' -m raw -a "dnf install -y python libselinux-python git; '
-                ' dnf update -y nettle" --user %s --ssh-common-args="%s" ' %
-                (self.vm.username, ' '.join(ssh_args)))
-            process.run(cmd, env=env)
-        
-        if distro == 'CentOS':
-            cmd = (' ansible localhost -i "localhost," --become '
-                ' -m raw -a "yum install -y python libselinux-python git; '
-                ' yum update -y nettle" --user %s --ssh-common-args="%s" ' %
-                (self.vm.username, ' '.join(ssh_args)))
-            process.run(cmd, env=env)
-        
-        # Parameters used to DEBUG mode
-        DEBUG = self.params.get('DEBUG', default=None)
-        iso_src = self.params.get('minishift_iso_src_path', default=None)
-        iso_dest = self.params.get('minishift_iso_dest_path', default=None)
-
-        # command use in the test from parameters
-        cmd_run = self.params.get('command', default=None)
+        # install python, git and update nettle to libvirt network
+        cmd = (' ansible localhost -i "localhost," --become '
+               ' -m raw -a "%s install -y python libselinux-python git; '
+               ' dnf update -y nettle" --user %s --ssh-common-args="%s" ' %
+               (self.pkg_mgm, self.vm.username, ' '.join(ssh_args)))
+        process.run(cmd, env=env)
 
-        if DEBUG is True:
+        if self.debug is True:
             # Create the directory to storage the ISO in VM
             cmd = (' ansible localhost -i "localhost," '
                    ' -m raw -a "mkdir -p %s" '
                    ' --user %s --ssh-common-args="%s" ' %
-                   (iso_src, self.vm.username, ' '.join(ssh_args)))
+                   (self.iso_src, self.vm.username, ' '.join(ssh_args)))
             process.run(cmd, env=env)
 
             # Copy the minishift.iso of host to VM
             cmd = (' ansible localhost -i "localhost," '
```

Comment:
ok, this one has to be independent, since it's using a different ansible module.